scripts/withdraw.js

```diff
@@ -53,16 +53,15 @@ const argv = require("yargs")
   })
   .version(false).argv
 
-const getAmount = async function(bracketAddress, tokenAddress, exchange) {
+const getAmount = async function(bracketAddress, tokenInfo, exchange) {
```

Comment:
I find the name of this function somewhat vague. What "amount" is being gotten here?